safe_transaction_service/account_abstraction/serializers.py

```diff
@@ -32,20 +32,26 @@
 # ================================================ #
 class SafeOperationSignatureValidatorMixin:
     """
-    Mixin class to validate SafeOperation signatures. _get_owners must be overridden to define
+    Mixin class to validate SafeOperation signatures. `_get_owners` can be overridden to define
```

Comment:
```suggestion
    Mixin class to validate SafeOperation signatures. 
```
A developer knows that he can override a function class :)

Reply:
Haha not always 😉